gpt4all-chat/modellist.cpp

```diff
@@ -1166,7 +1166,7 @@ void ModelList::updateModelsFromDirectory()
 
                 // All files that end with .bin and have 'ggml' somewhere in the name
                 if (((filename.endsWith(".bin") || filename.endsWith(".gguf")) && (/*filename.contains("ggml") ||*/ filename.contains("gguf")) && !filename.startsWith("incomplete"))
-                    || (filename.endsWith(".txt") && (filename.startsWith("chatgpt-") || filename.startsWith("nomic-")))) {
+                    || filename.endsWith(".rmodel")) {
```

Comment:
We have to be careful here not to break the current models / versions. We need to support backward compatibility with people's existing install/models that use the .txt file for chatgpt.